client/components/payment-activity/index.tsx

```diff
@@ -6,20 +6,53 @@ import { Card, CardBody, CardHeader } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 /**
- * Internal dependencies.
+ * Internal dependencies
  */
-import './style.scss';
+
+import EmptyStateAsset from 'assets/images/payment-activity-empty-state.svg?asset';
+import interpolateComponents from '@automattic/interpolate-components';
 import PaymentsActivityData from './payments-activity-data';
+import './style.scss';
 
 const PaymentActivity: React.FC = () => {
+	const { lifetimeTPV } = wcpaySettings;
+	const hasAtLeastOnePayment = lifetimeTPV !== 0;
```

Comment:
According to Dev tools, my account cache contents shows `(int) 0` but lifetimeTPV is `(string) "0"`
```
    'lifetime_total_payments_volume' => 0,
```
Not sure where this is getting cast as a string. Might need to cast to an `int` for this check or make the check less strict 
```suggestion
	const hasAtLeastOnePayment = lifetimeTPV != 0;
```

Reply:
I've just tested and it might work better with 
```suggestion
	const hasAtLeastOnePayment = lifetimeTPV > 0;
```
I've double-checked the server code and `lifetimeTPV` should never be negative. Refunds and disputes don't get taken into account for this value.

Reply:
Thanks for spotting this and finding a solution.  From [what I read](https://core.trac.wordpress.org/ticket/25280#comment:45) on a WP.org thread, it looks like `wp_localize_script` seems to enforce string type. The thread recommends usage of `wp_add_inline_script`. 

Also on the thread I found [this mentioned](https://core.trac.wordpress.org/ticket/25280#comment:34)

```
 The method supports objects/arrays, but not very well. The casting and decoding only happen on the root level, so while we're concerned about backwards-compatibility, the existing functionality is inconsistent.
```

I now nested `lifetimeTPV` within another property `paymentsActivity` and it renders as a number! 

Maybe that's why fees and other items nested within objects are preserving their number type. Probably why @elazzabi had also nested this within a `transactions` object initially. 

I did a quick try using `wp_add_inline_script` with json_decode ( since it needs the script verbatim as a string ). Somehow did not work. If this is the recommended way of doing things, it may require exploration. 

For now I am going with nesting `lifetimeTPV` to preserve it as a number.